packages/system-tests/scenarios/scaffolding.test.ts

```diff
@@ -248,7 +275,7 @@ describe('Empty VSCode workspace', () => {
     return await app.stop();
   });
 
-  it('Should not show any commands for non-SFDX workspace', async () => {
+  it('Should only show create project command for non-SFDX workspace', async () => {
```

Comment:
I assume this is internal, but it would be clearer if you moved "only" before "for".

Reply:
"Should show create project command **only** for non-SFDX workspace" is incorrect since it's designed to show up in SFDX workspaces and non-SFDX workspaces but it will be the only command to show up in a non-SFDX workspace. I wrote it like this to be a little bit more specific than "Should only show one command for non-SFDX workspace" but I can use that instead if that's more clear?